### ISSUE TITLE
[Enhancement] Disable temporary funcionality

### ISSUE THREAD

**Issue by JM63** [Seuleuzeuh/WPFTabTipMixedhardware#3]
Greetings
I have been using WPFTabTipMixeHardware for some time and it has worked very well, great job.

However, I would like to know if it is possible to deactivate its funcionality whenever I enter a certain window and reactivate it as soon as i leave that window.

Thanks,

**Comment by Seuleuzeuh** [comment]
Hi,

I will try to add this functionality asap.

I think i will do a new property IsEnabled (true by default), that will manage the Activation/Deactivation. 
You will have to set to false when you are in this certain window and set to true when you leave it. 
It will fit your behavior ?
Or did you want to exclude a list of window by their name (x:Name) ?

**Comment by JM63** [comment]
Hi,

I think IsEnabled will work perfectly and it will be a big improvement.

**Comment by Seuleuzeuh** [comment]
i've made a pre-released version of the functionnality in a [1.3.3-alpha package](https://www.nuget.org/packages/WPFTabTipMixedHardware/1.3.3-alpha).
Can you give it a try and tel me if it fit your need before i push my update ?

**Comment by JM63** [comment]
Ok I will test it ...

**Comment by Seuleuzeuh** [comment]
I've integrate/test it in my production app. In a particular behavior with our internal AutocompleteTextBox, i don't want the intenal re-focus call open the keayboard. I use the new property like this ;
```
WPFTabTipMixedHardware.Helpers.TabTip.Close();
//[...]
try
{
     WPFTabTipMixedHardware.TabTipAutomation.IsEnabled = false;
     Focus();
}
finally
{
      WPFTabTipMixedHardware.TabTipAutomation.IsEnabled = true;
}
```

I will publish the package in release version today, if you don't send a NO GO from your behavior/test.

**Comment by JM63** [comment]
I Sorry for my delayed response, but another project took me all the time.

This enhancement seems work well, the only situation that does not work is when the computer has no keyboard connected. Although IsEnabled = false the virtual keyboard is shown.

**Comment by Seuleuzeuh** [comment]
It's an OS behavior. 
I will try to force close it when OS launch it (with another property ForceCloseTabTib, false by default). 
I'll publish a released package when this update was include.

**Comment by Seuleuzeuh** [comment]
I've made the evolution. Added the AutoCloseTabTipWhenDisabled property (default true), who auto close all opening of the virtual keyboard (until IsEnable set to true or AutoCloseTabTipWhenDisabled set to false).

Open an other Issue if this does'nt work for your scenario, thanks.

**Comment by Seuleuzeuh** [comment]
Available in release [v1.3.4](https://github.com/Seuleuzeuh/WPFTabTipMixedhardware/releases/tag/v1.3.4)
